### DISCUSSION TITLE
20220117 Change in Volume Control with rotary encoder ?

### DISCUSSION THREAD

**Discussion by GDV0** [rern/rAudio#780] [Feature Requests]
Hello Rern,

Many thanks for the job you are doing on rAudio!

I was successfully using a rotary encoder to control the volume of a Bluetooth speaker but as of release 20220117 it seems to be no longer functional. (I'm using a Bluetooth speaker without volume control on it)
To be more precise:
With a Bluetooth speaker paired to rAudio
   - Play/Stop toggle is functional
   - Volume control is **not functional**

Without any Bluetooth speaker paired to rAudio:
   - Play/Stop toggle is functional
   - Volume control **is  functional**

FYI, I have the same issue with the latest version 20220122

It seems to be a new feature attached to the rotary encoder. If so, is it possible to make it optional ?

Thx
Gilles

**Comment by rern** [comment]
Please try:
- Addons > rAudio 1 
- long-press Update
- OK for Branch UPDATE

**Comment by GDV0** [comment]
Hi Rern

Thanks for this fast answer !
Unfortunately, I cannot notice any change after updating my RPi

Gilles

**Reply by rern** [reply]
Try the UPDATE again. It should be ok now.

**Comment by GDV0** [comment]
The change you have done seems to work perfectly

Thanks

**Comment by GDV0** [comment]
Hi rern

I performed additional tests and I have discovered this behavior:
First, rAudio is started without Bluetooth speaker. 
I can set the volume to any value using the rotary encoder (which is perfet)
Then I switch on my BT speaker. 
After successful pairing, the volume in rAudio  is forced to 100% and not maintained to the previous value (but as I wrote in my previous  now I can modify it !)

I was just a little bit surprised as rAudio was playing an AC/DC track when I switched on my BT speaker !!

Additional remark (January 28)
when paired to a BT speaker, it is possible to modify the volume using the buttons on the Web UI or the tap feature of the rotary widget but not when using the drag feature of the rotary widget
